### ISSUE TITLE
Proxy support

### ISSUE THREAD

**Issue by Commifreak** [xibosignage/xibo-linux#193]
**Describe the bug**
The snap seem not to support proxies...

I have embedded content. In this content, I want to display a image from outside out LAN. A proxy is needed for this. The windows client uses the system settings for that - that is working.

I cant force the client to use a proxy.

I tried:

* `http_proxy=http://x.x.x.x/ https_proxy=http://x.x.x.x/ xibo-player`
* Set Advanced -> Proxy Domain to http://x.x.x.x/ in xibo-player.options

**Logs**
I would - I dont know where they are stored.

**Environment (please complete the following information):**
- Linux Disto: Ubuntu 20.04 x64
- Linux Desktop Environment: GNOME
- How the player was launched: snap
- Player Version: 1.8-R5

**Comment by Commifreak** [comment]
This becomes more important currently. Simple Website integration doesnt work :(

**Comment by dasgarner** [comment]
I don't think this will be related to snap - it is more likely the web engine embedded by the player which does not support proxies.

We should investigate whether the proxy configured in the player options can be passed through into the web engine (which is what happens in windows).

**Comment by Commifreak** [comment]
That would be awesome!
